src/components/KTable/ColumnVisibilityMenu.vue

```diff
@@ -24,12 +24,25 @@
       </KTooltip>
 
       <template #items>
+        <KInput
+          v-model="searchColumnInMenu"
+          class="search-input"
+          placeholder="Search..."
```

Comment:
suggestion: Should the placeholder be more specific?

Reply:
Does `Search item` sound good ?

Reply:
Search columns?